backend/wmg/data/schemas/cube_schema.py

```diff
@@ -43,9 +41,11 @@
 )
 
 # The cube attributes that comprise the core data stored within the cube.
+filters_numeric = [tiledb.ByteShuffleFilter(), tiledb.ZstdFilter(level=+5)]
 expression_summary_logical_attrs = [
-    tiledb.Attr(name="nnz", dtype=np.uint64, filters=filters),  # TODO: Why uint64?
-    tiledb.Attr(name="sum", dtype=np.float32, filters=filters),
+    tiledb.Attr(name="nnz", dtype=np.uint64, filters=filters_numeric),  # TODO: Why uint64?
+    tiledb.Attr(name="sum", dtype=np.float32, filters=filters_numeric),
+    tiledb.Attr(name="sqsum", dtype=np.float32, filters=filters_numeric),
```

Comment:
adding sqsum to expression_summary cube - this is required for generating the default and fmg cubes (the latter two cubes are now being generated directly from the expression_summary cube).

adding new attributes only negligibly affects query times. considering that the expression summary cube is much smaller now after removing assay and development logical dimensions, query times will still be significantly faster compared to what they are currently.